src/components/HomePage/CreateWorkspaceModal/index.tsx

```diff
@@ -16,7 +20,7 @@ interface IWorkspaceAuthData {
   objectId: string;
 }
```

Comment:
Move this interface IWorkspaceAuthData to the interfaces.ts file of the Homepage folder.